### PR TITLE
feature - PLP Filter UI

### PR DESCRIPTION
**Pull request by Umasankar1522**

### CODE REVIEW

File: src/components/Accordian/Accordian.tsx [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+const Accordian = (props: Props): JSX.Element => {
+  const [expand, setExpand] = useState(true);
   return (
     <div className={styles.accordian}>
-        <div onClick={(): void => setExpand(!expand)} className={expand ? styles.accordian_textChange : styles.accordian_text  }>
-          {expand === false ? <FaPlus className={styles.accordian_icon_space}/>:<FaMinus className={styles.accordian_icon_space}/>}
+      <div className={styles.accordian__label + ' ' + `${expand ? styles.accordian_textChange : styles.accordian_text}`}>
+        <div onClick={(event): void => { event.stopPropagation(); setExpand(!expand); }}>
+          {expand === false ? <FaPlus className={styles.accordian_icon_space} /> : <FaMinus className={styles.accordian_icon_space} />}
           {props.childone}
```
Comment (prateekoli9) [inline src/components/Accordian/Accordian.tsx:19]: Rename prop to title

File: src/components/FilterComponent/FilterItem/FilterItem.component.tsx [hunk 2]
```diff
@@ -1,32 +1,86 @@
...
+      </div>
+    </div>
+  );
+};
+
+const FilterInputs: FC<any> = ({ data, onInputChecked, checkedCount }: any): JSX.Element => {
+  const dispatchAction = useDispatch();
+  const [checked, setChecked] = useState<boolean>(false);
+  const handleCheckBox = (event: any): void => {
+    console.log('onChange Triggered', event.target.value);
```
Comment (prateekoli9) [inline src/components/FilterComponent/FilterItem/FilterItem.component.tsx:34]: Remove console.log

File: src/components/FilterComponent/Filter.tsx [hunk 3]
```diff
@@ -1,19 +1,78 @@
...
+});
+
+const Filter = (): JSX.Element => {
+  const textFacets = useSelector((state) => state.product.textFacets);
+  const [filterItems, setFilterItems] = useState([]);
+  const [mount, setMount] = useState(true);
+  const dispatchAction = useDispatch();
+
+  const onClearAll = (): void => {
+      setMount(false);
```
Comment (prateekoli9) [inline src/components/FilterComponent/Filter.tsx:32]: Remove code related to this mount state variable, handle it using internal state of FilterComp

File: src/components/FilterComponent/Filter.tsx [hunk 4]
```diff
@@ -1,19 +1,78 @@
...
-        </div> 
-        {filterList.map((val)=>
-           <FilterItem key={Math.random()} data={val}/> 
-        )}
-    </div> 
+      <h2 className={styles.filterComponent__heading}>
+        <span className={styles.filterComponent__heading__spanFilter}>FILTER  BY</span>
+        <span className={styles.filterComponent__heading__spanClear} onClick={onClearAll}> CLEAR ALL </span>
+      </h2>
+      {mount && filterItems.map((val, index) => <FilterItem key={index} data={val} /> )}
```
Comment (prateekoli9) [inline src/components/FilterComponent/Filter.tsx:51]: remove mount

Comment (Umasankar1522) [reply src/components/FilterComponent/Filter.tsx]: removed mount state and its usage over the code

File: src/components/FilterComponent/Filter.tsx (hunk 3 above)
Comment (Umasankar1522) [reply src/components/FilterComponent/Filter.tsx]: removed mount state and its usage over the code

File: src/components/FilterComponent/FilterItem/FilterItem.component.tsx (hunk 2 above)
Comment (Umasankar1522) [reply src/components/FilterComponent/FilterItem/FilterItem.component.tsx]: removed console.log() statements

File: src/components/Accordian/Accordian.tsx (hunk 1 above)
Comment (Umasankar1522) [reply src/components/Accordian/Accordian.tsx]: updated the Accordion as common component

**Pull request merged**